config-example.yaml

```diff
@@ -45,8 +50,10 @@ ca: #ACME and none ACME CA
       url: https://acme-v02.api.letsencrypt.org/directory
```

Comment:
Is this the ACME endpoint? Or the CA page send to the client? Or both?

Reply:
This is the endpoint used by the acme plugin, but it is also provided over the API in the field "url".

Reply:
Both, ok.

`url` from the API CA scheme is there to route the user to a CA landing page and not to the ACME endpoint.

So we need a config option for the ACME API endpoint. I will open an issue...